project.clj

```diff
@@ -65,15 +65,14 @@
                  ;; bridge to allow some spring/activemq stuff to log over slf4j
                  [org.slf4j/jcl-over-slf4j "1.7.10"]
                  ;; WebAPI support libraries.
-                 [net.cgrand/moustache "1.1.0" :exclusions [ring/ring-core org.clojure/clojure]]
-                 [compojure "1.3.3"]
+                 [compojure "1.4.0"]
```

Comment:
I think another patch/ticket is a good idea, we definitely want to close the book on this soon though.